index.js

```diff
@@ -3,7 +3,13 @@
 
 module.exports = {
   name: 'ember-cli-airbrake',
+  isEnabled: function () {
```

Comment:
This isn't checking the configuration in config/environment but instead the value passed to EmberApp in the users Brocfile when they build. This allows them to either enable or disable it independently of it being configured.

N.b, this should be a separate PR really. Been messy and haven't worked on a branch. Can break up if you want.